x-pack/plugins/data_usage/server/services/autoops_api.ts

```diff
@@ -164,7 +164,6 @@ export class AutoOpsAPIService {
           ...requestConfig.httpsAgent.options,
           cert: requestConfig.httpsAgent.options.cert ? 'REDACTED' : undefined,
           key: requestConfig.httpsAgent.options.key ? 'REDACTED' : undefined,
-          ca: requestConfig.httpsAgent.options.ca ? 'REDACTED' : undefined,
```

Comment:
__question__: Do you have a plan for how you’re going to test this integration with AutoOps locally (to iterate quicker), on CI and MKI (to make sure it's not broken i.e. as a quality gate)?